#! /bin/sh /usr/share/dpatch/dpatch-run
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disable the blurb about replying to a bug via email
## DP: at least until this is made to work.

@DPATCH@
diff -urNad gforge~/common/tracker/Artifact.class.php gforge/common/tracker/Artifact.class.php
--- gforge~/common/tracker/Artifact.class.php	2006-06-12 10:22:31.000000000 +0200
+++ gforge/common/tracker/Artifact.class.php	2006-06-12 15:43:49.000000000 +0200
@@ -1206,16 +1206,16 @@
 			$changes=array();
 		}
 		
-		$body = $this->ArtifactType->getName() ." item #". $this->getID() .", was opened at ". date( $sys_datefmt, $this->getOpenDate() ). 
-			"\nYou can respond by visiting: ".
-			"\nhttp://".$GLOBALS['sys_default_domain']."/tracker/?func=detail&atid=". $this->ArtifactType->getID() .
-				"&aid=". $this->getID() .
-				"&group_id=". $this->ArtifactType->Group->getID() .
-			"\nOr by replying to this e-mail entering your response between the following markers: ".
-			"\n".ARTIFACT_MAIL_MARKER.
-			"\n(enter your response here)".
-			"\n".ARTIFACT_MAIL_MARKER.
-			"\n\n".
+		$body = $this->ArtifactType->getName() ." item #". $this->getID() .", was opened at ". date( $sys_datefmt, $this->getOpenDate() ). "\n".
+// 			"\nYou can respond by visiting: ".
+// 			"\nhttp://".$GLOBALS['sys_default_domain']."/tracker/?func=detail&atid=". $this->ArtifactType->getID() .
+// 				"&aid=". $this->getID() .
+// 				"&group_id=". $this->ArtifactType->Group->getID() .
+// 			"\nOr by replying to this e-mail entering your response between the following markers: ".
+// 			"\n".ARTIFACT_MAIL_MARKER.
+// 			"\n(enter your response here)".
+// 			"\n".ARTIFACT_MAIL_MARKER.
+// 			"\n\n".
 			$this->marker('status',$changes).
 			 "Status: ". $this->getStatusName() ."\n".
 			$this->marker('priority',$changes).
